src/status_im/ui/screens/signing/views.cljs

```diff
@@ -90,14 +92,81 @@
     [react/view {:padding 6}
      [react/text {:style {:color colors/blue}} (i18n/label :t/cancel)]]]])
 
-(views/defview password-view [{:keys [type error in-progress? enabled?]}]
+(views/defview keycard-pin-view []
+  (views/letsubs [pin [:hardwallet/pin]]
+    [react/view
+     [pin.views/pin-view
+      {:pin           pin
+       :retry-counter nil
+       :step          :sign
+       :status        nil
+       :error-label   nil}]]))
+
+(defn- keycard-connect-view []
+  [react/view {:padding-vertical 20
+               :flex             1
+               :align-items      :center
+               :justify-content  :center}
+   [react/image {:source      (resources/get-image :keycard-phone)
+                 :resize-mode :center
+                 :style       {:width  160
+                               :height 170}}]
+   [react/view {:margin-top 10}
+    [react/text {:style {:text-align  :center
+                         :color       colors/gray
+                         :font-size   15
```

Comment:
This is default typography, so not necessary to set both font-size and line-height